api/ordinals/src/api/middleware/metrics.ts

```diff
@@ -0,0 +1,93 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { ApiMetrics } from '../../metrics/metrics';
+
+export function registerMetricsMiddleware(fastify: FastifyInstance, metrics: ApiMetrics) {
```

Comment:
Hey Alin, apologies, I should've explained this yesterday. The `prom-client` already takes care of per-request timing measurements and all kinds of default stats, so you shouldn't worry about adding these to your metrics.

The same goes for DB queries, because those get measured by another tool we use called PgHero.

You should only focus on making sure the prometheus metrics are being applied correctly to this API and the Runes API via the use of `prom-client`.